frontend/src/components/elements/GraphVizChart/GraphVizChart.tsx

```diff
@@ -62,9 +63,18 @@ class GraphVizChart extends React.PureComponent<Props> {
       // Override or reset the graph height
       if (this.props.element.get("height")) {
         graph.height(this.props.element.get("height"))
+      } else if (this.props.height) {
```

Comment:
So the height from the user takes precedence over the height in the UI? 

How does fullscreen mode override it?

Reply:
Yes! I did it in that way for all the components. (I copied the same behavior from width property)